monkey/monkey_island/windows/create_certificate.bat

```diff
@@ -20,13 +20,13 @@ copy "%mydir%windows\openssl.cfg" "%mydir%bin\openssl\openssl.cfg"
 
 :: Change file permissions
 SET adminsIdentity="BUILTIN\Administrators"
-FOR /f %%O IN ('whoami') DO SET ownIdentity=%%O
+FOR /f "delims=''" %%O IN ('whoami') DO SET ownIdentity=%%O
 
 FOR %%F IN ("%mydir%cc\server.key", "%mydir%cc\server.csr", "%mydir%cc\server.crt") DO (
 
 	:: Remove all others and add admins rule (with full control)
-	echo y| cacls %%F" /p %adminsIdentity%:F
+	echo y| cacls %%F" /p "%adminsIdentity%":F
```

Comment:
This change isn't needed.
Why is there a `"` after `%%F`? I don't remember there being a particular reason for that. I think it might have been a typo, overlooked at the time. Can you test the script without it and remove it if it works fine?

Reply:
Worked good.